azure-pipelines.yml

```diff
@@ -216,18 +226,25 @@ jobs:
       displayName: "Run tests"
       workingDirectory: $(Build.BinariesDirectory)/build
 
-    - task: CMake@1
```

Comment:
Why is this a script and not two steps?

Reply:
I'm not sure what you're referring to specifically?
each `- script:` or `- task:` is a step.

Reply:
Sorry, was mobile. Harder to type.

This looks like it shifts from a `cmake` specific task, to a script that invokes `cmake` and then `cp`. I'm curious about the change there.

Reply:
Ah I see. Well, it's just to be able to group operations, instead of having a step just for a `cp`.

Reply:
Seems fine, mostly I'm curious.